src/server/dcp_server/models.py

```diff
@@ -248,11 +260,15 @@ def train(self, imgs, masks):
         self.segmentor.train(imgs, masks_instances)
         # create patch dataset to train classifier
         masks_classes = list(masks[:,1,...]) #[((mask > 0) * np.arange(1, 4)).sum(-1) for mask in masks]
-        patches, labels = create_patch_dataset(imgs,
+        patches, patch_masks, labels = create_patch_dataset(imgs,
                                                masks_classes,
                                                masks_instances,
                                                noise_intensity = self.train_config["classifier"]["train_data"]["noise_intensity"],
                                                max_patch_size = self.train_config["classifier"]["train_data"]["patch_size"])
+        
+        if self.classifier_class == "RandomForest":
+            patches = create_dataset_for_rf(patches, patch_masks)
```

Comment:
can we rename patches to something more explicit? here this is now features not patches so it is confusing.